docs/source/contributing/redux.md

```diff
@@ -10,19 +10,22 @@ myst:
 # Redux
 
 ```{note}
-This documentation is a work in progress. Any help is welcome to fill in the
-gaps!
+This documentation is a work in progress.
+
+Any help is welcome to fill in the gaps!
 ```
 
 As with any other complex React project, the way global state is handled across
-all components has a big impact on the overall architecture. Basic knowledge of
-{term}`Redux` is needed to understand this part, but Volto's use of Redux is
-"typical" and you can find plenty of examples in Volto's code base.
+all components has a big impact on the overall architecture.
+
+Basic knowledge of {term}`Redux` is needed to understand this part, but Volto's
+use of Redux is "typical" and you can find plenty of examples in Volto's code
+base.
```

Comment:
You're still wrapping sentences to a line length. We use one sentence per line.

We also don't need one sentence per paragraph by having two line breaks between sentences. We need to preserve the paragraphs.

```suggestion
Basic knowledge of {term}`Redux` is needed to understand this part, but Volto's use of Redux is "typical" and you can find plenty of examples in Volto's code base.
```